packages/ember-auto-import/ts/webpack.ts

```diff
@@ -163,10 +171,6 @@ export default class WebpackBundler implements BundlerHook {
       use: {
         loader: 'babel-loader-8',
         options: {
-          // do not use the host project's own `babel.config.js` file
-          configFile: false,
-          babelrc: false,
```

Comment:
Why are these removed? As far as I can tell, we need them still (or we run much more than `@babel/preset-env` on deps that are imported).

Reply:
This directly reverts the fixes landed in https://github.com/ef4/ember-auto-import/pull/342... 🤔

Reply:
Submitted https://github.com/ef4/ember-auto-import/pull/360 to bring back the changes from #342. Happy to discuss over there...